scripts/v1alpha2/run-cleanpodpolicy-all.sh

```diff
@@ -50,3 +50,12 @@ echo "Running mnist test"
 MNIST_TEST_IMAGE_TAG="pytorch-dist-mnist_test:1.0"
 go run ./test/e2e/v1alpha2/cleanpolicy_all.go --namespace=${NAMESPACE} --image=${REGISTRY}/${MNIST_TEST_IMAGE_TAG} --name=mnistjob-cleanall
 
```

Comment:
do we need to test cleanpod policy with mpi images also?  I feel it is not needed